.github/workflows/ci.yml

```diff
@@ -118,15 +118,13 @@ jobs:
         id: 'install-monorepo'
         with:
           install: '${{ matrix.projectName }}...'
-          build: 'false'
           pull-playwright-cache: ${{ matrix.testEnv.shouldCreate && matrix.testType == 'e2e' }}
 
       - uses: './.github/actions/setup-woocommerce-monorepo'
-        if: ${{ github.ref_type != 'tag' }}
+        if: ${{ github.ref_type != 'tag' && matrix.testType != 'unit:php' }}
```

Comment:
Should we try applying same approach for `api` tests?

Reply:
API tests are passing as well with the same approach.

Reply:
API should required the same build as e2e tests.

Reply:
API tests are REST API tests, correct @adimoldovan ?